.claude/agents/helpdot-inline-reviewer.md

```diff
@@ -68,6 +68,17 @@ mcp__github_inline_comment__create_inline_comment:
   body: "**Terminology violation**: Use 'workspace' instead of 'policy' to match Expensify standards."
```

Comment:
Since here we are using the single quotes inside of the comments, is it correct to use the single quote around the the content?